### DISCUSSION TITLE
Why a can't get feature and lables in data loader, in x_trian y_train

### DISCUSSION THREAD

**Discussion by tonyhaya** [mrdbourke/pytorch-deep-learning#293] [Q&A]
In chapter 1 and 2, the data come in diferent variable, however, in the chapter 3, when I need to train a model, all answer and question (feature and label) are in the same place, Why the model keep learning, shoun't run some error. My model runs. One more question why can't use outside of `for enumerate()`, to get the data in different local

`Train_data=torchvision.datasets.MNIST(root="Train data",train=True, #Do we want the training dataset
    download=True,#do we want to download y/n
    transform=torchvision.transforms.ToTensor(),#how do we want to transform the data
    target_transform=None)`

`Test_data=torchvision.datasets.MNIST(root="dTest ata",train=False, #Do we want the training dataset
    download=True,#do we want to download y/n
    transform=torchvision.transforms.ToTensor(),#how do we want to transform the data
    target_transform=None)`

Obs: Always visualize
![image](https://user-images.githubusercontent.com/111157701/217908435-108a3eda-d789-480b-a90c-da8422e5250c.png)
